server/routes/controllers/emailController.js

```diff
@@ -1,19 +1,17 @@
-const sendgrid = require('@sendgrid/mail')
-
 const bindFunctions = require('../../bindFunctions')
 const { emailConfig } = require('../../config')
 
 const emailController = {
-  _init (emailClient = sendgrid) {
+  _init () {
     bindFunctions(this)
-
-    this.emailClient = emailClient
-    this.emailClient.setApiKey(emailConfig.sendgridApiKey)
+    this.emailClient = {
+      send: emailConfig.getSend()
+    }
```

Comment:
instead of having this logic here in the controller could the email config return the emailClient? this way we can still expect to pass `_init()` the emailClient (defaulting to the emailClient pulled from the config), and be able to mock it in the test below as well

what do you think?